using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -613,36 +861,46 @@ Here are some examples in Objective-C and Swift:
 **Objective-C**
 
 ```objectivec
-[ACPTarget clearPrefetchCache];
+[ACPTarget resetExperience];
 ```
 
 **Swift**
 
 ```swift
-ACPTarget.clearPrefetchCache()
+ACPTarget.resetExperience()
 ```
 {% endtab %}
 
 {% tab title="React Native" %}
-### clearPrefetchCache
+
+#### Syntax
+
+```javascript
+resetExperience()
+```
+
+#### Example
 
 **JavaScript**
 
 ```javascript
-ACPTarget.clearPrefetchCache();
+ACPTarget.resetExperience();
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Retrieve Location Content requests
+## retrieveLocationContent
+
+This API to sends a batch request to the configured Target server for multiple mbox locations. 
+
+The main difference with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#loadrequests) API is in usage with prefetch APIs. For a prefetched mbox, unlike `loadRequests` API, invoking this API does not send a display notification to the configured Target server. If you are not prefetching mbox content, then there is no difference with `loadRequests` API for sending batch request to Target.
 
-Use this API to send a batch request to the configured Target server for multiple mbox locations. The main difference with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#loadrequests) API is in usage with prefetch APIs. For a prefetched mbox, unlike `loadRequests` API, invoking this API does not send a display notification to the configured Target server. If you are not prefetching mbox content, then there is no difference with `loadRequests` API for sending batch request to Target.
+For mbox locations in the target requests list that are not already prefetched, this API sends a batch request to the configured Target server. The content for the mbox locations that have been prefetched in a previous request are returned from the SDK, and no additional network request is made. Each target request object in the list contains a callback function, which is invoked when content is available for its given mbox location.
```

Comment:
Please reword section so that it reads as follows:
"For mbox locations in the Target requests list that have not already been prefetched, this API sends a batch request to the configured Target server. The content for the mbox locations that have been prefetched in a previous request are returned from the SDK, and no additional network request is made. Each Target request object in the list contains a callback function, which is invoked when content is available for its given mbox location."